MusicDB/1 table.sql

```diff
@@ -0,0 +1,56 @@
+Use MusicDB
+go
+drop table if exists instrument
+go 
+create table dbo.instrument(
+InstrumentID int not null identity primary key,
+CustomerFirstName varchar(15) not null
+    constraint ck_Instrument_Customer_first_name_cannot_be_blank check (CustomerFirstName > ''), 
+CustomerLastName varchar(15) not null
+    constraint ck_Instrument_Customer_last_name_cannot_be_blank check (CustomerLastName > ''), 
+Instrument varchar(20) not null
+    constraint ck_Instrument_Instrument_cannot_be_blank check (Instrument > ''), 
+Category varchar(15) not null
+    constraint ck_Instrument_Category_cannot_be_blank check (Category > ''),
+ID as concat(substring(Instrument, 1, 3), (substring(Category, 1,3))) persisted, 
+MonthlyRentalFee dec (4,2) not null, 
```

Comment:
-- LL This decimal will only allow fees up until $99.. although most will probably be under that price, you should make your decimal allowance at least let up to $999